 Camera: Added null check for memory reference

Null memory reference in releaseRecordingFrame will cause
crash in cfuzz tool.
Added Null check for memory reference

diff --git a/services/camera/libcameraservice/device1/CameraHardwareInterface.h b/services/camera/libcameraservice/device1/CameraHardwareInterface.h
index 2ffd184..ef2e6e0 100644
--- a/services/camera/libcameraservice/device1/CameraHardwareInterface.h
+++ b/services/camera/libcameraservice/device1/CameraHardwareInterface.h
@@ -363,6 +363,10 @@ class CameraHardwareInterface : public virtual RefBase {
     void releaseRecordingFrame(const sp<IMemory>& mem)
     {
         ALOGV("%s(%s)", __FUNCTION__, mName.string());
+        if (mem == NULL) {
+             ALOGE("%s: NULL memory reference", __FUNCTION__);
+             return;
+        }
         if (mDevice->ops->release_recording_frame) {
             ssize_t offset;
             size_t size;
